packages/appsync-modelgen-plugin/src/__tests__/visitors/appsync-swift-visitor.test.ts

```diff
@@ -132,11 +142,17 @@ describe('AppSyncSwiftVisitor', () => {
       public struct snake_case: Model {
         public let id: String
         public var name: String?
+        public var createdAt: Temporal.DateTime?
+        public var updatedAt: Temporal.DateTime?
         
         public init(id: String = UUID().uuidString,
-            name: String? = nil) {
+            name: String? = nil,
+            createdAt: Temporal.DateTime? = nil,
```

Comment:
should these even be in the constructor? since they are managed by apppsync I shouldnt be able to set them, only read them?